baiji/config.py

```diff
@@ -32,13 +41,20 @@ def load(self):
 
         raw_data = {}
 
-        if os.path.isfile(os.path.expanduser(self.aws_cli_path)):
-            import ConfigParser
-            aws_cli_config = ConfigParser.ConfigParser()
-            aws_cli_config.read([os.path.expanduser(self.aws_cli_path)])
+        # load credentials
+        if os.path.isfile(os.path.expanduser(self.aws_credentials_path)):
+            aws_credential_config = self._load_aws_config_file(self.aws_credentials_path, {})
+
+            raw_data.update({
+                'AWS_ACCESS_KEY': aws_credential_config.get('default', 'aws_access_key_id'),
+                'AWS_SECRET': aws_credential_config.get('default', 'aws_secret_access_key'),
+            })
+
+        # load settings
+        if os.path.isfile(os.path.expanduser(self.aws_settings_path)):
+            aws_settings_config = self._load_aws_config_file(self.aws_settings_path, {'region': 'us-east-1'})
```

Comment:
Yeah boto's `connect_to_region` function required a region and it should be a standalone config. The reason I'm providing default here is the config parse will actually raise an error if it's not in the `default` section. Same applies to the access keys and secrets actually but I assume those are required so an exception is expected if the `aws_access_key_id` is missing.